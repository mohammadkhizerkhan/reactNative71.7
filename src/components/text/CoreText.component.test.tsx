import React from 'react';
import {render, screen} from '@testing-library/react-native';
import CoreText from './CoreText.component';

describe('should haveText Component', () => {
  it('render text correctly', () => {
    const Text = render(<CoreText>hello</CoreText>);
    expect(screen.queryByText('hello')).not.toBeNull();
    expect(screen.queryByText('hello'));
    expect(Text.getByText('hello')).toBeTruthy();
    expect(Text).toMatchSnapshot();
  });
});
