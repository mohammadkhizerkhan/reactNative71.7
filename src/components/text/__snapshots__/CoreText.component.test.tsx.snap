// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should haveText Component render text correctly 1`] = `
<Text
  allowFontScaling={false}
  style={
    [
      {
        "fontSize": 16,
        "letterSpacing": -0.17,
      },
      undefined,
    ]
  }
>
  hello
</Text>
`;
